src/test/java/csv/FileServiceImplTest.java

```diff
@@ -0,0 +1,96 @@
+package csv;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class FileServiceImplTest {
+
+    @Rule
+    public ExpectedException exceptionReadRule = ExpectedException.none();
+
+    @Rule
+    public ExpectedException exceptionWriteRule = ExpectedException.none();
+
+    @Test
+    public void testReadFile_OK() {
+        String inputCsvFile = "inputTestFile.csv";
+
+        try (BufferedWriter fileBufferedWriter = new BufferedWriter(new FileWriter(inputCsvFile))) {
+            fileBufferedWriter.write("type,fruit,quantity" + System.lineSeparator());
+            fileBufferedWriter.write("b,fruit1,1" + System.lineSeparator());
+            fileBufferedWriter.write("s,fruit1,1" + System.lineSeparator());
+            fileBufferedWriter.write("r,fruit1,1" + System.lineSeparator());
+            fileBufferedWriter.write("p,fruit1,1" + System.lineSeparator());
+            fileBufferedWriter.write("b,fruit2,2" + System.lineSeparator());
+            fileBufferedWriter.write("s,fruit2,2" + System.lineSeparator());
+            fileBufferedWriter.write("r,fruit2,2" + System.lineSeparator());
+            fileBufferedWriter.write("p,fruit2,2" + System.lineSeparator());
+        } catch (IOException e) {
+            throw new RuntimeException("Can't open output file" + inputCsvFile,e);
+        }
+
+        FileServiceImpl csvFileService = new FileServiceImpl();
+        List<String> inFileStringList = csvFileService.readFile(inputCsvFile);
+        Assert.assertEquals("firstRow read failed",inFileStringList.get(0),"b,fruit1,1");
+        Assert.assertEquals("firstRow read failed",inFileStringList.get(3),"p,fruit1,1");
+        Assert.assertEquals("firstRow read failed",inFileStringList.get(7),"p,fruit2,2");
+    }
+
+    @Test
+    public void testReadFile_Exception() {
```

Comment:
name of the test method should be like this:
<name_of_the_method_you_test>_<special_state_if_you_have_one>_<notOk/ok - is it ok or not ok situation>
For example: 
readFile_nonExistentFile_notOk
fix naming of test methods in all classes